modules/rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly make `book` and `Person` to have Getters and Setters. Kindly make the modification to meet the condition of a rental belongs to a book and also belongs to a person.